lib/core/repository/videos/public_website.rb

```diff
@@ -0,0 +1,25 @@
+module Core::Repository
+  module Videos
+    class PublicWebsite < Core::Repository::Base
+      def initialize
+        self.connection = Core::Registry::Connection[:public_website]
+      end
+
+      def find(id)
+        response = connection.get('/%{locale}/videos/%{id}.json' %
```

Comment:
Same as I mentioned before, I think stylistically it's easier to parse each argument on a separate line.